starter-code/app.js

```diff
@@ -1,18 +1,26 @@
-const express        = require('express');
-const path           = require('path');
-const favicon        = require('serve-favicon');
-const logger         = require('morgan');
-const cookieParser   = require('cookie-parser');
-const bodyParser     = require('body-parser');
+const express = require('express');
+const path = require('path');
+const favicon = require('serve-favicon');
+const logger = require('morgan');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
 const expressLayouts = require('express-ejs-layouts');
-const mongoose       = require('mongoose');
-const index          = require('./routes/index');
-const users          = require('./routes/users');
-const drones         = require('./routes/drones');
+const mongoose = require('mongoose');
+const index = require('./routes/index');
+const users = require('./routes/users');
+const drones = require('./routes/drones');
 
 mongoose.connect('mongodb://localhost/drones-dev');
 
 const app = express();
+// layouts
+
+mongoose.Promise = Promise;
+mongoose.connect('mongodb://localhost/drones', {
```

Comment:
I you establish the connection with this (well done), get rid of line 13. You don't need to connect two times. Furthermore, you are connecting to two different databases.